src/identity/interaction/email-password/handler/InteractionHandler.ts

```diff
@@ -1,5 +1,5 @@
 import type { KoaContextWithOIDC } from 'oidc-provider';
-import type { HttpRequest } from '../../../../server/HttpRequest';
+import type { Operation } from '../../../../ldp/operations/Operation';
```

Comment:
Maybe LDP is no longer the right namespace then.

Reply:
True, this has actually been the case for several things in that folder for some time. Now we're just reaching the point where it pretty is no longer unique for most things in there. So we can probably just rename the folder to something more generic (which seems an easier transition than moving everything in it around). `core` perhaps.